docs/articles/intro/deployment-scenarios.md

```diff
@@ -0,0 +1,389 @@
+		}
+	}
+}
+
+```
+
+Typically you use a very lightweight `ActorSystem` inside ASP.NET applications, and offload heavy-duty work to a separate Windows Service via Akka.Remote / Akka.Cluster.
+
+### Interaction Between Controllers and Akka.NET
+
+In the sample below, we use an Web API Controller:
+
+```csharp
+[ApiController]
+[Route("[controller]")]
+public class WriteApiController : ControllerBase
+{
+        
+   private readonly ILogger<WriteApiController> _logger;
+   private readonly IActorBridge _bridge;
+
+   public WriteApiController(ILogger<WriteApiController> logger, IActorBridge bridge)
+   {
+      _logger = logger;
+      _bridge = _bridge;
+   }
+
+   [HttpPost]
+   [Route("post")]
+   public async Task<IActionResult> Post([FromBody] object data)
+   {
+       _bridge.Tell(data);
+	   //you can use Ask if you need response from the Actor
+       //await _bridge.Ask<T>(data);
+       return Ok();
+   }
+}
+```
+
+### Wire up Akka.NET and ASP.NET Core in `Startup.cs`
+
+We need to replace the default `IHostedService` with `AkkaService` and also inject `IActorBridge` and we do that with `Startup.cs`
+
+```csharp
+public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            
+            services.AddControllers();
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "Akka.NET", Version = "v1" });
+            });
+
+            // creates instance of IPublicHashingService that can be accessed by ASP.NET
+            services.AddSingleton<IActorBridge, AkkaService>();
+
+            // starts the IHostedService, which creates the ActorSystem and actors
+            services.AddHostedService<AkkaService>(sp => (AkkaService)sp.GetRequiredService<IActorBridge>());
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseSwagger();
+                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Akka.Net v1"));
+            }
+
+            app.UseRouting();
+
+            //app.UseAuthorization();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
+        }
+    }
+```
+
+Visit our site's blog post for [Best Practices for Integrating Akka.NET with ASP.NET Core and SignalR](https://petabridge.com/blog/akkadotnet-aspnetcore/)
+
+## Windows Service
```

Comment:
Emphasize Headless services, not Windows Services - Headless Service is the deployment class, Windows Service is a specific way of running and managing them.

Installing a Headless Service as a Windows Service should be mentioned in passing but there's abundant literature on how to do that inside Microsoft's documentation.

Reply:
Noted